index.html

```diff
@@ -0,0 +1,261 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"></script>
+    <link href="css/styles.css" rel="stylesheet">
+
+    <title>Yannick-Noel Aka | Portfolio</title>
+
+</head>
+
+<body>
+
+    <header>
+        <nav class="nav-bar">
+            <div class="logo">
+                <a href="#">My Logo</a>
+            </div>
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <div class="myMenu">
+                <ul class="nav-list">
+                    <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+                    <li><a href="#about" class="list-item">About</a></li>
+                    <li><a href="#contact" class="list-item">Contact</a></li>
+                </ul>
```

Comment:
[ ] While this project looks just like the given design but on the desktop mode it's giving us the menu items in vertical line while it should be horizontal. On desktop mode, the hamburger should be gone. 
 
<img width="938" alt="Capture" src="https://user-images.githubusercontent.com/58504655/128181850-4a7bd556-0529-42f3-9faa-38f5feca44e3.PNG">

Reply:
* done 👍